test/Conversion/TorchToLinalg/elementwise.mlir

```diff
@@ -94,3 +94,89 @@ func.func @insufficient_dims_for_triu(%arg0: !torch.vtensor<[?],f32>) -> !torch.
   %0 = torch.aten.triu %arg0, %int0 : !torch.vtensor<[?],f32>, !torch.int -> !torch.vtensor<[?],f32>
   return %0 : !torch.vtensor<[?],f32>
 }
+
+// -----
+
+// CHECK-LABEL:   func.func @torch.aten.quantize_per_tensor(
```

Comment:
per our testing policy do not include these tests: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

Reply:
Understood, thanks for the link! I will remove those tests.